### PR TITLE
Fixed type hint

### PR DESCRIPTION
**Pull request by root-aza**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /**
  * Checks if the given value equals eleven.
+ * 
+ * @param int value
```
Comment (Snegirekk) [inline index.js:4]: We cannot be sure what the given value is int

Comment (root-aza) [reply index.js:4]: Зато стат анализ гарантирует

Comment (Snegirekk) [reply index.js:4]: What if I'd like to check what the string "eleven" is equals to integer 11?

**Comment by piemekanika** [comment]
outdated

File: index.js (hunk 1 above)
Comment (root-aza) [reply index.js:4]: Кастуй

**Pull request closed without merging**